osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SecretManagementServiceTest.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
```

Comment:
Copyright for new files should be for Alliander N.V. instead of Smart Society Services B.V.